workspaces/announcements/README.md

```diff
@@ -0,0 +1,35 @@
+# Announcements plugin for Backstage
+
+## Overview
```

Comment:
Can we please add a note to this about the history of this plugin, looking for the ownership changes. If that is somewhere else please point that out, this is a big PR and I may have missed it 👍